_includes/sections/working-here/hiring-new-people.md

```diff
@@ -1,342 +1,199 @@
+other social networks.
+
+The second way we find people is by advertising. We advertise jobs on
+Stack Overflow, GitHub, WorkInStartups and Unicorn Hunt. Not all jobs go on all
+of these boards - we pick whatever seems most appropriate for the job.
 
-Communication is a two-way process and it is important to check understanding
-and confidence, and value the other person in decision-making. Explaining the
-task and its objectives clearly means people know what’s expected of them and
-the part they play. It empowers them to take responsibility and own how they
-approach the work.
+The third way we find people is by making sure we're regularly blogging about
+what we're doing, and being open about our culture, work and process. We accept
+[general applications](https://www.dxw.com/careers/general-application/) from
+people who are interested in dxw but who don't fit an open job.
 
-###### Recognise achievement and celebrate success
+#### Review applications
 
-A simple ‘thank you’ shows someone that what they have done is appreciated, and
-is hugely motivating. Celebrating major, and minor, achievements is an important
-part of our culture at dxw. It builds team spirit and morale.
+All applications arrive via Workable. We review these applications as a team to
+decide who to take forward.
 
-###### Guard our values
+An application review is a very quick process. We look at the information given
+by the candidate and decide if it is likely that the applicant has the skills,
+experience and personal qualities that we need. If it is likely, we progress
+them to the next stage.
+
+#### Screening
 
-Lead by example. The actions and behaviours of line managers must always
-demonstrate our dxw values. We should always hold ourselves accountable to our
-values, and encourage the same in all of our colleagues, including those we line
-manage.
+The purpose of the screening is to:
 
-###### Continuous constructive feedback
+1. Introduce ourselves and explain a little more about the job and life at dxw
+1. Talk to the candidate about their experience and explore why they are
+   interested in the job
+1. Decide whether you think they are quite likely to have the experience and
+   personal qualities that we are looking for
+1. Get an idea of the candidate's
+   salary expectation
 
-Find out how each of your team members prefers to receive feedback, and develop
-an approach which reflects that. Ongoing positive and constructive feedback is
-essential if someone is to feel valued, learn from their experiences and become
-better at what they do.
+Try to keep the conversation quite high-level, and avoid going into too much
+technical detail. This part of the process is more about experience and personal
+qualities than about skills. Things to keep in mind:
 
-###### Be honest
+* Does this person seem likely to embody our [values](#our-values)?
+* Does this person have an interest in our [mission](#why-are-we-all-here)?
```

Comment:
This is the only bit that was lost from the previous version. `#why-are-we-all-here` doesn't exist anymore, and the closest replacement was `#who-we-are`.

Reply:
I've replaced Why we are all here with a link to Who we are and add a link to our Start with people principle.